src/main/java/xyz/nucleoid/extras/integrations/game/StatisticsIntegration.java

```diff
@@ -69,11 +70,10 @@ private void handleStatisticsBundle(GameSpace space, String namespace, GameStati
         body.add("bundle", bundleObject);
         body.addProperty("game_id", gameId.toString());
         this.sendBundle(body);
-        // TODO: Enable this when the web thing is done
-//        space.getPlayers().sendMessage(new TranslatableText("text.nucleoid_extras.statistics.web_url", gameId)
-//                .formatted(Formatting.GRAY, Formatting.ITALIC)
-//                .styled(style -> style.withClickEvent(new ClickEvent(ClickEvent.Action.OPEN_URL,
-//                        "https://nucleoid.xyz/game/#" + gameId))));
+        space.getPlayers().sendMessage(new TranslatableText("text.nucleoid_extras.statistics.web_url", gameId)
+                .formatted(Formatting.GRAY, Formatting.ITALIC)
+                .styled(style -> style.withClickEvent(new ClickEvent(ClickEvent.Action.OPEN_URL,
+                        "https://stats.nucleoid.xyz/games/" + gameId))));
```

Comment:
It’s not very important- can we have the URL sent from the backend? 🙂

Reply:
Maybe, but at the minute the backend doesn't really have a way of sending a 'reply' to a message, so maybe when its refactored at some point